src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -305,10 +305,10 @@ private Dictionary<int, int> ComputeMethodBodyUsage(MetadataReader mr, StreamRea
     private void CreateTrimmedAssembly(PEReader peReader, string trimmedAssemblyFilePath, FileStream fs, Dictionary<int, int> methodBodyUses)
     {
         using FileStream os = File.Open(trimmedAssemblyFilePath, FileMode.Create);
-        using MemoryStream memStream = new MemoryStream((int)fs.Length);
 
         fs.Position = 0;
-        fs.CopyTo(memStream);
+        fs.CopyTo(os);
+        fs.Flush();
```

Comment:
you don't need to flush, this happens as part of disposing the FileStream anyway